src/resources/views/crud/columns/upload_multiple.blade.php

```diff
@@ -10,7 +10,7 @@
     @if ($value && count($value))
         @foreach ($value as $file_path)
         @php
-            $column['wrapper']['href'] = $column['wrapper']['href'] ?? ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
+            $column['wrapper']['href'] = ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
```

Comment:
Hmm I understand the problem, but wouldn't this prevent the `href` feature from ever being overwritten for the `upload_multiple` column type?

Maybe we can find a way to fix the bug, but keep the user ability to overwrite the `href`. What if we define the `href` above, in the `@php` block, if it wasn't already set, as a closure? I don't know...

Reply:
I agree with you.

Or you mean something: `$column['wrapper']['href'] = $column['wrapper']['href'] ?? function($disk, $prefix, $file_path) { return ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) ); }`

And in this line we do `$link = is_callable($column['wrapper']['href']) ? $column['wrapper']['href']($disk, $prefix, $file_path) : $column['wrapper']['href'];` ?

Let me know what you think so I can update the PR accordingly.

Best,
Pedro